integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/smooks/SmooksIntegrationTestCase.java

```diff
@@ -64,28 +63,29 @@ public void close() throws Exception {
             resourceAdminServiceClient.deleteResource("/_system/config/smooks_config.xml");
         } finally {
             super.cleanup();
-            Thread.sleep(3000);
             resourceAdminServiceClient = null;
         }
     }
 
     @SetEnvironment(executionEnvironments = {ExecutionEnvironment.STANDALONE
 })
     @Test(groups = {"wso2.esb"}, description = "Sending a Large File To Smooks Mediator")
-    public void testSendingToSmooks() throws IOException, EndpointAdminEndpointAdminException,
-                                             LoginAuthenticationExceptionException,
-                                             XMLStreamException, InterruptedException {
-
-        File afile = new File(getClass().getResource(File.separator + "artifacts" + File.separator + "ESB" + File.separator + "synapseconfig" + File.separator + "smooks" + File.separator + "person.csv").getPath());
-        File bfile = new File(getClass().getResource(File.separator + "artifacts" + File.separator + "ESB" + File.separator + "synapseconfig" + File.separator + "smooks" + File.separator).getPath() + "test" + File.separator + "in" + File.separator + "person.csv");
-
-        FileUtils.copyFile(afile, bfile);
-        Thread.sleep(40000);
-
-        File outfile = new File(getClass().getResource(File.separator + "artifacts" + File.separator + "ESB" + File.separator + "synapseconfig" + File.separator + "smooks" + File.separator).getPath() + "test" + File.separator + "out" + File.separator + "Out.xml");
-
-        String smooksOut = FileUtils.readFileToString(outfile);
-        Assert.assertTrue(smooksOut.contains("<csv-record number=\"160\"><firstname>Andun</firstname><lastname>Sameera</lastname><gender>Male</gender><age>4</age><country>SriLanka</country></csv-record>"));
+    public void testSendingToSmooks() throws Exception {
+        String smooksResourceDir = getClass().getResource(
+                File.separator + "artifacts" + File.separator + "ESB" + File.separator + "synapseconfig"
+                        + File.separator + "smooks" + File.separator).getPath();
+        Path source = Paths.get(smooksResourceDir, "person.csv");
+        Path destination = Paths.get(smooksResourceDir + "test", "in", "person.csv");
+
+        Files.createDirectories(Paths.get(smooksResourceDir, "test", "in"));
+        Files.copy(source, destination, StandardCopyOption.REPLACE_EXISTING);
+        Thread.sleep(2000);
```

Comment:
if we cannot avoid Thread sleep here shall we add a comment mentioning "for what we are waiting for"